blazar/policies/networks.py

```diff
@@ -53,7 +53,22 @@
                 'method': 'DELETE'
             }
         ]
-    )
+    ),
+    policy.DocumentedRuleDefault(
+        name=POLICY_ROOT % 'get_allocations',
```

Comment:
The handler defined above references a policy `list_allocations`, and here it is `get_allocations` - they should agree. I think it should be called `list_allocations`, as the special case (listing for a given segment) is just the list operation with a filter applied.

Reply:
I think I edited this when debugging and didn't push. Hosts has the only upstream code on this. It uses 'get_allocations' for both. I figure we'll keep it consistent for now and refactor both at some point.